feat(curriculum): New steps to teach lists (#55167)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/66694fc4bba24f33ca01fa5b.md

---
id: 66694fc4bba24f33ca01fa5b
title: Step 3
challengeType: 20
dashedName: step-3
---

# --description--

Once you have a list, most often you can't add items directly to it changing the declaration.

You can add an item to a list using the `append` method. For example:

> 

Reviewer: What if we ask to print the list after appending the new item?

---
id: 66694fc4bba24f33ca01fa5b
title: Step 3
challengeType: 20
dashedName: step-3
---

# --description--

> Once you have a list, most often you can't add items directly to it changing the declaration.

Reviewer 2: ```suggestion
Once you have a list, you generally won't add items to it by changing the declaration.
```

Not sure what we are trying to say here, tbh.

Reviewer 3: I assume it is trying to explain that you do not add new items by re-declaring/re-initializing the list with new values. Although, it seems like an odd, or incomplete, preamble to teaching list methods.

I might suggest directly going into the methods without the preamble.

> Python has a handful of list methods. Such as methods for adding or removing list items.
